CMakeLists.txt

```diff
@@ -1,36 +1,42 @@
-cmake_minimum_required(VERSION 2.8)
-project(Kalman)
+cmake_minimum_required(VERSION 3.0)
+project(Kalman VERSION 0.1.0 LANGUAGES CXX)
+
+include(CTest)
+
+set(KALMAN_DEFINITIONS "-std=c++11")
```

Comment:
Ya, I agree. I was just using the naming convention I had seen in other *Config.cmake files, but I'vs also seen names like FOO_CXX_FLAGS so thats the style I've now changed it to.